### PR TITLE
Added Feature and Task issue templates

### PR DESCRIPTION
**Pull request by adamczykba**
This change is [Reviewable](https://reviewable.io/reviews/servicecatalog/oscm-rest-api/32)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/task.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+labels: task
+assignees: ''
+
+---
+
+**Summary**
+Clear description or/and business justification of what the task is about.
+
+**Details**
+If applicapable, detailed information regarding things to be considered, requirements etc.
```
Comment (Crystalzord) [inline .github/ISSUE_TEMPLATE/task.md]: Misspelled. Should be "applicable".

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+labels: feature request
+assignees: ''
+
+---
+
+**Summary**
+Clear description or/and business justification of what the feature is about.
+
+**Details**
+If applicapable, detailed information regarding things to be considered, requirements etc.
```
Comment (Crystalzord) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Misspelled. Should be "applicable".

File: .github/ISSUE_TEMPLATE/task.md (hunk 1 above)
Comment (adamczykba) [reply .github/ISSUE_TEMPLATE/task.md]: @Crystalzord Good call. In that case anyway it's also misspelled on main oscm repo

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (adamczykba) [reply .github/ISSUE_TEMPLATE/feature_request.md]: @Crystalzord ditto

**Pull request merged**